js/wp-seo-metabox.js

```diff
@@ -7,699 +7,92 @@
-		jQuery( '.wpseo-metabox-tabs' ).show();
-		// End Tabs code
-
-		var cache = {}, lastXhr, focuskwHelpTriggered = false;
-
-		jQuery( '#' + wpseoMetaboxL10n.field_prefix + 'focuskw' ).autocomplete( {
-				minLength: 3,
-				formatResult: function( row ) {
-					return jQuery( '<div/>' ).html( row ).html();
-				},
-				source: function( request, response ) {
-					var term = request.term;
-					if ( term in cache ) {
-						response( cache[ term ] );
-						return;
-					}
-					request._ajax_nonce = wpseoMetaboxL10n.wpseo_keyword_suggest_nonce;
-					request.action = 'wpseo_get_suggest';
-
-					lastXhr = jQuery.getJSON( ajaxurl, request, function( data, status, xhr ) {
-							cache[ term ] = data;
-							if ( xhr === lastXhr ) {
-								response( data );
-							}
-						}
-					);
-				}
-			}
-		);
-
-		jQuery( '#' + wpseoMetaboxL10n.field_prefix + 'title' ).keyup( function() {
-				ystUpdateTitle();
-			}
-		);
-		jQuery( '#title' ).keyup( function() {
-				ystUpdateTitle();
-				ystUpdateDesc();
-			}
-		);
-		jQuery( '#parent_id' ).change( function() {
-				ystUpdateTitle();
-				ystUpdateDesc();
-			}
-		);
-		// DON'T 'optimize' this to use descElm! descElm might not be defined and will cause js errors (Soliloquy issue)
-		jQuery( '#' + wpseoMetaboxL10n.field_prefix + 'metadesc' ).keyup( function() {
-				ystUpdateDesc();
-			}
-		);
-
-		// Set time out because of tinymce is initialized later then this is done
-		setTimeout(
-			function() {
-				ystUpdateSnippet();
-
-				// Adding events to content and excerpt
-				if ( typeof tinyMCE !== 'undefined' && tinyMCE.get( 'content' ) !== null ) {
-					tinyMCE.get( 'content' ).on( 'blur', ystUpdateDesc );
-				}
-
-				if ( typeof tinyMCE !== 'undefined' && tinyMCE.get( 'excerpt' ) !== null ) {
-					tinyMCE.get( 'excerpt' ).on( 'blur', ystUpdateDesc );
-				}
-			},
-			500
-		);
-
-		jQuery( document ).on( 'change', '#' + wpseoMetaboxL10n.field_prefix + 'focuskw', function() {
-				var focuskwhelpElm = jQuery( '#focuskwhelp' );
-				if ( jQuery( '#' + wpseoMetaboxL10n.field_prefix + 'focuskw' ).val().search( ',' ) !== -1 ) {
-					focuskwhelpElm.click();
-					focuskwHelpTriggered = true;
-				}
-				else if ( focuskwHelpTriggered ) {
-					focuskwhelpElm.qtip( 'hide' );
-					focuskwHelpTriggered = false;
-				}
-
-				ystUpdateSnippet();
-			}
-		);
-
-		jQuery( '.yoast_help' ).qtip(
-			{
-				content: {
-					attr: 'alt'
-				},
-				position: {
-					my: 'bottom left',
-					at: 'top center'
-				},
-				style: {
-					tip: {
-						corner: true
-					},
-					classes: 'yoast-qtip qtip-rounded qtip-blue'
-				},
-				show: 'click',
-				hide: {
-					fixed: true,
-					delay: 500
-				}
-
-			}
-		);
-	}
-);
+        /*
+         * Used for caching the replaced vars so we only do an AJAX request for each var once
+         * Ignore for JSHint as it is used but JSHint doesn't properly reflect that
+         */
+        if ( jQuery( '.wpseo-metabox-tabs-div' ).length > 0 ) {
```

Comment:
This logic should be split into multiple functions, currently it is very hard to read and to find out what is happening.